src/main/kotlin/io/bazel/kotlin/builder/cmd/BUILD.bazel

```diff
@@ -39,9 +39,12 @@ kt_bootstrap_binary(
         "-XX:-MaxFDLimit",
     ],
     main_class = "io.bazel.kotlin.builder.cmd.Build",
-    runtime_library = ":build_lib",
     shade_rules = "//src/main/kotlin:shade.jarjar",
     visibility = ["//src:__subpackages__"],
+    runtime_deps = [
+        ":build_lib",
+        "@bazel_tools//tools/jdk:JacocoCoverage",
```

Comment:
This is the actual fix. The `BUILD.bazel` version of this file has this included as a runtime dependency already.